machine_test.go

```diff
@@ -781,6 +801,18 @@ func TestCaptureFifoToFile(t *testing.T) {
 	}
 	defer os.Remove(fifoPath)
 
+	fifoLogPath := fifoPath + ".log"
+	fifo, err := os.OpenFile(fifoLogPath, os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0644)
+	if err != nil {
```

Comment:
nit: You've got `assert` in this file already, so you might as well do `assert.NoError(t, err, "failed to create destination file")`.